hedera-node/src/main/java/com/hedera/services/sigs/metadata/StateChildrenSigMetadataLookup.java

```diff
@@ -177,7 +178,8 @@ public SafeLookupResult<ScheduleSigningMetadata> scheduleSigningMetaFor(
 		if (linkedRefs != null) {
 			linkedRefs.link(id.getScheduleNum());
 		}
-		final var schedule = stateChildren.schedules().get(EntityNum.fromScheduleId(id));
+		final var schedule = stateChildren.schedules().byId().get(
```

Comment:
It would be nice to hide the `.byId().get(new EntityNumVirtualKey(...` implementation details in a `schedules().getById(id)` method---[law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter#In_object-oriented_programming) and so on.

Reply:
Would it be ok if we left these like this? It would be a major re-factor to change at this point and it basically would just be creating more boilerplate. All this is abstracted behind `HederaScheduleStore` for the most part.

Reply:
True, I think this kind of boilerplate helps the reader's eye keep moving tho.